db/migrate/20230503183426_create_entities.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEntities < ActiveRecord::Migration[7.0]
+  def change
+    create_table :entities do |t|
+      t.string :name
+      t.integer :author_id
+      t.decimal :amount
+
+      t.timestamps
```

Comment:
- [OPTIONAL] The database schema should reflect the following structure:

![image](https://user-images.githubusercontent.com/79058364/227439921-57baa020-2dfb-42d7-9613-d4a21a80577a.png)

- [OPTIONAL] According to the Entity relational database Diagram, there is a many to many relationship between Group and Entity so, You should have created a join table to link the two tables. Kindly address this 💯 😉 
- Note: In Rails and PostgreSQL, creating a join table is necessary for modeling many-to-many relationships because it allows us to establish a connection between two tables without duplicating data.